test/tests/markers/reporter_point_marker/tests

```diff
@@ -0,0 +1,20 @@
+[Tests]
+  group = 'Reporter Point Marker'
+  design = "/Markers/index.md /ReporterPointMarker.md"
+  issues = '#18886'
+  [mark_only]
+    type = 'Exodiff'
+    input = 'point_marker_test.i'
+    exodiff = 'point_marker_test_out.e'
+    scale_refine = 2
+    requirement = "The adaptivity system shall create an auxiliary field variable that marks "
+                  "elements containing the points from the reporter for refinement."
+  []
+  [wrong_size_error]
+    type = 'RunException'
+    input = 'point_marker_test.i'
+    cli_args = 'Adaptivity/Markers/active=bad_coord'
+    expect_err = "The coordinate vectors are a different size."
```

Comment:
If you insist on outputting the individual vector sizes in that error message then please include them in the expected error string!